pkg/manager/member/tikv_upgrader.go

```diff
@@ -190,6 +190,11 @@ func (u *tikvUpgrader) upgradeTiKVPod(tc *v1alpha1.TidbCluster, ordinal int32, n
 func (u *tikvUpgrader) readyToUpgrade(upgradePod *corev1.Pod, tc *v1alpha1.TidbCluster) bool {
 	evictLeaderTimeout := tc.TiKVEvictLeaderTimeout()
 	tlsEnabled := tc.IsTLSClusterEnabled()
+	if tc.TiKVStsActualReplicas() < 2 {
```

Comment:
What about we checking the `len(podOrdinals)` in https://github.com/pingcap/tidb-operator/blob/86ea694ef0/pkg/manager/member/tikv_upgrader.go#L95 and just update the partition and return if the length < 2?